.github/workflows/test.yml

```diff
@@ -49,15 +49,22 @@ jobs:
             **/**.go
             go.mod
             go.sum
+      # Install cross compiler for ARM64. Export CC env variable.
+      - name: Install compiler for arm64.
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y gcc-aarch64-linux-gnu
+          echo "CC=aarch64-linux-gnu-gcc" >> $GITHUB_ENV
+        if: matrix.go-arch == 'arm64'
       - name: Build ibc-go
-        run: GOARCH=${{ matrix.go-arch }} LEDGER_ENABLED=false make build
+        run: GOARCH=${{ matrix.go-arch }} CGO_ENABLED=1 LEDGER_ENABLED=false make build
```

Comment:
`CGO_ENABLED` is, by default, disabled if targeting an arch different than native.